From ff33f536350b62c872e1893021273f0ec93094e3 Mon Sep 17 00:00:00 2001
From: Xubao Jiang <xubaoj@codeaurora.org>
Date: Mon, 26 Apr 2021 18:20:45 +0800
Subject: [PATCH] Disable close_range

close_range first appeared in Linux 5.9.
Disable it as it causes boot up issue with linux-msm 5.4

This reverts commit b8cfa2da7c103d612b9608ade580c0a41538450c.

Upstream-Status: Inappropriate [disable feature]

Signed-off-by: Xubao Jiang <xubaoj@codeaurora.org>
---
 src/basic/fd-util.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/basic/fd-util.c b/src/basic/fd-util.c
index 07a7b3a..0addf6d 100644
--- a/src/basic/fd-util.c
+++ b/src/basic/fd-util.c
@@ -216,7 +216,7 @@ static int cmp_int(const int *a, const int *b) {
 }
 
 int close_all_fds(const int except[], size_t n_except) {
-        static bool have_close_range = true; /* Assume we live in the future */
+        static bool have_close_range = false;
         _cleanup_closedir_ DIR *d = NULL;
         struct dirent *de;
         int r = 0;
-- 
2.7.4

